### PR TITLE
Update README for better, more precise steps

### PR DESCRIPTION
**Pull request by drohit-cb**
## Description Of Change

This PR helps update the README to better provide more accurate steps for anyone to come and follow the quickstart guide.

## Testing Procedure

Reviewed the readme from my branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,4 +262,4 @@ Thanks for considering contributing to the project! Please refer to [our contrib
 
 ## Documentation
 
-There are numerous examples in the [`examples directory`](./examples) to help get you started. For even more, refer to our [documentation website](https://docs.cdp.coinbase.com/) for detailed definitions, API specifications, integration guides, and more!
+There are numerous examples in the [`examples directory`](./examples) to help get you started. For even more, refer to our [documentation website](https://docs.cdp.coinbase.com/staking/docs/welcome) for detailed definitions, API specifications, integration guides, and more!
```
Comment (ProfMoo) [inline README.md:265]: Thanks for making the link update 👍

File: README.md [hunk 2]
```diff
@@ -18,10 +18,21 @@ A traditional infrastructure-heavy staking integration can take months. Coinbase
...
 
-1. Install this package: `npm install @coinbase/staking-client-library-ts`
+1. Navigate to your project directory where you would like to set up this client, and install it via:
+   ```shell
+   npm install @coinbase/staking-client-library-ts
+   ```
 2. Create and download an API key from the [Coinbase Developer Platform](https://portal.cdp.coinbase.com/access/api).
 3. Place the key named `.coinbase_cloud_api_key.json` at the root of this repository.
-4. Run one of the code samples [below](#stake-partial-eth-💠) or any of our [provided examples](./examples/) :rocket:.
+4. Install necessary tools/packages to get typescript files compiled and running:
```
Comment (ProfMoo) [inline README.md:27]:
```suggestion
4. Install necessary Typescript dependencies:
```

**Review by marcin-cb — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -18,10 +18,21 @@ A traditional infrastructure-heavy staking integration can take months. Coinbase
 
 Prerequisite: [Node 20+](https://www.npmjs.com/package/node/v/20.11.1)
 
-1. Install this package: `npm install @coinbase/staking-client-library-ts`
+1. Navigate to your project directory where you would like to set up this client, and install it via:
```
Comment (ProfMoo) [inline README.md:21]:
```suggestion
1. In a fresh directory, run:
```

**Review by ProfMoo — COMMENTED** [review]
Tested it out myself from a fresh dir and it works great 😄. Just a few small comments.

File: README.md (hunk 3 above)
Comment (drohit-cb) [reply README.md:21]: ya makes sense - don't mind pulling this change in, but my idea was developers could be integrating this into existing projects. But assuming they are going through a quick start, they just want to try something quick - so a fresh dir is a good idea.

**Pull request merged**